core/lib/server/stages.ts

```diff
@@ -15,134 +14,6 @@ import type { AutoReturnType } from "../types";
-				jsonArrayFrom(
-					eb
-						.selectFrom("move_constraint")
-						.whereRef("move_constraint.destinationId", "=", "stages.id")
-						.selectAll("move_constraint")
-				).as("moveConstraintSources"),
-				jsonArrayFrom(
-					eb
-						.selectFrom("PubsInStages")
-						.select("pubId")
-						.whereRef("stageId", "=", "stages.id")
-						.select(pubValuesByRef("pubId"))
-				).as("pubs"),
-				jsonArrayFrom(
-					eb
-						.selectFrom("permissions")
-						.innerJoin("_PermissionToStage", "permissions.id", "_PermissionToStage.A")
-						.whereRef("stages.id", "=", "stages.id")
-						.selectAll("permissions")
-						.select((eb) => [
-							jsonObjectFrom(
-								eb
-									.selectFrom("members")
-									.whereRef("members.id", "=", "permissions.memberId")
-									.selectAll("members")
-									.select((eb) => [
-										jsonObjectFrom(
-											eb
-												.selectFrom("users")
-												.select([
-													"users.id",
-													"users.firstName",
-													"users.lastName",
-													"users.avatar",
-													"users.email",
-												])
-												.whereRef("members.userId", "=", "users.id")
-										)
-											.$notNull()
-											.as("user"),
-									])
-							).as("member"),
-							jsonObjectFrom(
-								eb
-									.selectFrom("member_groups")
-									.selectAll("member_groups")
-									.whereRef("member_groups.id", "=", "permissions.memberGroupId")
-									.select((eb) => [
-										jsonArrayFrom(
-											eb
-												.selectFrom("users")
-												.innerJoin(
-													"_MemberGroupToUser",
-													"member_groups.id",
-													"_MemberGroupToUser.B"
-												)
-												.whereRef("users.id", "=", "_MemberGroupToUser.A")
-												.select([
-													"users.id",
-													"users.firstName",
-													"users.lastName",
-													"users.avatar",
-													"users.email",
-												])
-										).as("users"),
-									])
-							).as("memberGroup"),
-						])
-				).as("permissions"),
-				jsonArrayFrom(
-					eb
-						.selectFrom("integration_instances")
-						.whereRef("integration_instances.stageId", "=", "stages.id")
-						.selectAll()
-						.select((eb) => [
-							jsonObjectFrom(
-								eb
-									.selectFrom("integrations")
-									.selectAll("integrations")
-									.whereRef(
-										"integrations.id",
-										"=",
-										"integration_instances.integrationId"
-									)
-									.$narrowType<{ actions: JsonValue }>()
-							)
-								.$notNull()
-								.as("integration"),
-						])
-						.$narrowType<{ config: JsonValue }>()
-				).as("integrationInstances"),
-				jsonArrayFrom(
-					eb
-						.selectFrom("action_instances")
-						.whereRef("action_instances.stageId", "=", "stages.id")
-						.selectAll()
-						.$narrowType<{ config: JsonValue }>()
-				).as("actionInstances"),
-			])
-			.selectAll("stages")
-			.orderBy("order asc")
-	);
```

Comment:
this query is unused (superseded by a later function in this file) so I deleted it instead of fixing it

Reply:
I think that's fine, I don't think we intend to use it in the short term